components/content/Icon/Icon.ts

```diff
@@ -0,0 +1,35 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { JSXCustomElement } from '../../../types/jsx-custom-element';
+
+export interface IconAttributes {
+  shape: 'default' | 'circle';
+}
+
+@customElement('diamond-icon')
+export class Icon extends LitElement {
+  @property({ reflect: true }) shape?: IconAttributes['shape'];
+
+  render() {
+    return html`
+      <div part="icon" aria-hidden="true">
```

Comment:
This would ignore the aria-label added to an SVG, no?

Reply:
Yeah, I think aria label should be on interactive elements only and we always ignore the icons